javascript/builtins/AggregateError.json

```diff
@@ -173,7 +173,7 @@
             },
             "status": {
               "experimental": true,
-              "standard_track": false,
+              "standard_track": true,
```

Comment:
Can we add a link to [this text fragment](https://html.spec.whatwg.org/multipage/structured-data.html#:~:text=if%20value%20has%20an%20%5B%5Berrordata%5D%5D%20internal%20slot):
```json
"spec_url": "https://html.spec.whatwg.org/multipage/structured-data.html#:~:text=if%20value%20has%20an%20%5B%5Berrordata%5D%5D%20internal%20slot",
```

Reply:
I'm not a big fan of these text fragment URLs as I can't validate them (https://github.com/mdn/browser-compat-data/pull/23958) but I will not stay in the way if we think they are adding value.

Reply:
In our October 29, 2024 meeting we reached consensus to use text fragments where anchors aren't available.

In this case, we could link to the section instead, but it wouldn't be as straight-forward.

Reply:
In my opinion, any text fragment spec link is better than none.